src/styles/foundation/_focus-ring.scss

```diff
@@ -27,6 +29,21 @@
   }
 }
 
+@mixin focus-ring-animated($size: 'base', $show: 0) {
+  $easing: cubic-bezier(0.4, 0.22, 0.28, 1);
```

Comment:
I wasn't sure what to do with these easing values. Wonder if we keep them apart like this until we know the final functions needed in the system?

Reply:
And the focus ring around the checkbox and radio are animated which is why I added this mixin.